Decompiler/PscDecompiler.cpp

```diff
@@ -1379,11 +1379,6 @@ void Decompiler::PscDecompiler::dumpBlock(size_t startBlock, size_t endBlock)
             }
             m_Log << '\n';
         }
-        DumpTree tree([&] (std::ostream& line)
```

Comment:
The removed parts of the trace logging should probably be added back in future commits, such as what was removed here.